### PR TITLE
Spec cleanup

### LINKED ISSUE #174: Improve specification handling
find a better way to capture specifications. use non-dated version of specs, use JSON and group specs by SDO. This info can be useful in combination with the conformance framework.

### PR DESCRIPTION
**Pull request by podborski**
close #174

### CODE REVIEW

File: src/components/DataDisplay/Table.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ export default function Table({ data, globalFilter }: { data: object[]; globalFi
                 return columnHelper.accessor(header, {
                     // eslint-disable-next-line react/no-danger, react/no-unstable-nested-components
                     cell: (info) =>
-                        info.getValue() && (
+                        info.getValue() && ( // TODO: only add a hyperlink if the specification MPEG entry is true
```
Comment (podborski) [inline src/components/DataDisplay/Table.tsx:72]: @DenizUgur hi Deniz, could you maybe help with this one? I would like to make sure that only MPEG specs are forwarded to FFC. I already got rid of the specifications.csv and replaced it with a json which has the MPEG boolean field. I think this is now pretty easy to add.

**Comment by DenizUgur** [comment]
Yep, I'll create a PR today.

File: src/components/DataDisplay/Table.tsx (hunk 1 above)
Comment (DenizUgur) [reply src/components/DataDisplay/Table.tsx:72]: See #190

**Comment by podborski** [comment]
@DenizUgur thanks for the PR. The only small issue I see is that it does not work when you search.

**Comment by DenizUgur** [comment]
> @DenizUgur thanks for the PR. The only small issue I see is that it does not work when you search.

Sorry for that, #191 should fix it

**Pull request merged**